WF-507:Dogfooding fixes (#7)
spatialos/FlexibleProjectExample: README.md

## Logs location
The logs are stored in `logs` subdirectory of the location where your `spatialos.json` file is. For example, if my `spatialos.json` file is in `/workspace/project/SpatialOS`, the SpatialOS logs will go to `/workspace/project/SpatialOS/logs`.

## Known Issues

> * `spatial local worker launch` is not supported.

Reviewer: `spatial local worker launch` is not supported. You will need to launch workers manually through the steps indicated in (link to the part of the doc above)

?

Author: Re: How do we start workers in the future: Yes, users will need to manually start them.
Re: Should this be a known issue: Moved this note into a FAQ which will later become part of our migration guide.

> * `spatial project history snapshot convert` is not supported.

Author: Moved to the FAQ page.

> * For [client workers](docs/reference/client-worker-configuration.md), it is currently not possible to specify the base command and command line arguments, with which a worker is started.

Reviewer: This seems related to point one. Maybe say something like how you just simply need to run the base command and command line arguments yourself instead of using a json?

Author: Removed this point altogether.

> * [Client workers](docs/reference/client-worker-configuration.md) that are not built using Unity or Unreal, while supported by the flexible project structure in general, are not supported by our [Launcher](https://docs.improbable.io/reference/latest/shared/operate/launcher#the-launcher) yet. As a result, you will not be able to start the client worker provided in this repository from the Launcher, if you deploy this project in the cloud.

Reviewer: Nit: This point is a bit wordy, I think we can remove the `while supported by the flexible project structure in general,` part.

> * [Client workers](docs/reference/client-worker-configuration.md) that are not built using Unity or Unreal, while supported by the flexible project structure in general, are not supported by our [Launcher](https://docs.improbable.io/reference/latest/shared/operate/launcher#the-launcher) yet. As a result, you will not be able to start the client worker provided in this repository from the Launcher, if you deploy this project in the cloud.

Author: Shortened sentence.

## Logs location
The logs are stored in `logs` subdirectory of the location where your `spatialos.json` file is. For example, if my `spatialos.json` file is in `/workspace/project/SpatialOS`, the SpatialOS logs will go to `/workspace/project/SpatialOS/logs`.

## Known Issues

> * Our [Launcher](https://docs.improbable.io/reference/latest/shared/operate/launcher#the-launcher) currently only supports Unity and Unreal based [client workers](docs/reference/client-worker-configuration.md). As a result, you will not be able to start the client worker provided in this repository from the Launcher, if you deploy this project in the cloud.

Reviewer 2: ```suggestion
* Our [Launcher](https://docs.improbable.io/reference/latest/shared/operate/launcher#the-launcher) currently only supports Unity and Unreal based [client-workers](docs/reference/client-worker-configuration.md). As a result, if you deploy this project to the cloud, you can't start the provided client-worker from the Launcher.
```

Author: Is `client-worker` (with the `-` in the middle) a convention?

Reviewer 2: Yes, server-worker and client-worker are conventions we (tech writers) are trying to push for throughout the docs.

You might also have noticed my adding "type" or "instance" in a couple of places - this is another thing we hope to make consistent, but it's a bit harder because it involves going through every single mention of "worker" and deciding what exactly we're talking about... so for the moment we have to live with some inconsistency.

Author: Applied your suggestion.

[Server worker configuration](docs/reference/server-worker-configuration.md)

[Client worker configuration](docs/reference/client-worker-configuration.md)

## Migration Guide

> [FAQ](docs/migration-guide/faq.md)

Reviewer 2: Are we planning to add a migration guide to replace this at some point?

Author: The plan is to have a `migration guide` section containing multiple articles.
